### PR TITLE
Updated Typescript

### PR DESCRIPTION
**Pull request by Peeterush**
`update-index` broke in 930167cf013131bec8ed769d70f16b1528135a61 because the old lock file uses v20.3.0 of `@types/node`. Generating a new lock file updates it to v22.8.2, resolving the issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
         "eslint-plugin-license-header": "0.6.1",
         "jsdom": "25.0.1",
         "set-value": "4.1.0",
-        "typescript": " 5.5.4",
+        "typescript": "^5.6.3",
```
Comment (dlemstra) [inline package.json:60]: I am using only the exact version in the `package.json` file. Can you change that?

Comment (Peeterush) [reply package.json]: Right, of course!

**Comment by Peeterush** [comment]
I'm currently looking into that yes. The errors can be ignored by adding a `@ts-expect-error` comment before each of the lines, but I'm wondering if TS is actually catching a bug here. Going by the error messages it seems that among others `Channels.Meta0` is the exact same value as `Channels.Meta32`. Is that intentional?

**Comment by dlemstra** [comment]
WebAssembly is only 32 bit so those channels should be removed.

**Comment by dlemstra** [comment]
I also had to delete my `node_modules` folder to get the update to `v22.8.2`.

**Comment by dlemstra** [comment]
Just pushed a commit where I recreated the file with node `22.11`.

**Pull request merged**